### PR TITLE
Add MacOS stuff

### PR DESCRIPTION
**Pull request by WeixuanZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,29 @@ your package manager...?
...
 
+A better solution is to install Windows Linux Subsystem (WSL), which is a full
+Linux kernel in a virtual machine. (If you think this is too much, just use Git Bash) 
+
+You may also be interested in downloading the _Windows Terminal_ from Microsoft Store.
+
 **MacOS**
 
-Ask @WeixuanZ or something lol. Can someone PR?
+The must-have package manager on MacOS is [Homebrew](https://brew.sh), which you can install by
```
Comment (tim-clifford) [inline README.md:23]:
```suggestion
The must-have package manager on MacOS is [Homebrew](https://brew.sh), which
you can install by
```
79 chars please :)

File: README.md [hunk 2]
```diff
@@ -124,9 +144,8 @@ rm -rf .git/modules/repo-name
 **BEWARE: files deleted on a terminal will not go to the recycle bin**
 
 N.B. Launching Git Bash from Windows puts you in "C:\\Users\\username" (written
-as `~`) when you start it. Launching a linux terminal usually puts you in
-"/home/username" (also written as `~`). Ask @WeixuanZ to explain MacOS or
-something lol.
+as `~`) when you start it. Launching a linux or MacOS terminal usually puts you in
+"/home/username" (also written as `~`).
```
Comment (tim-clifford) [inline README.md:148]:
```suggestion
as `~`) when you start it. Launching a linux or MacOS terminal usually puts you
in "/home/username" (also written as `~`).
```

File: README.md [hunk 3]
```diff
@@ -13,9 +13,29 @@ your package manager...?
...
+
+Then you can install packages through
+
+```sh
+brew install <package>  # suggestions: git (of course), python, wget...
+```
+
+> If you encounter a permission error, be cautious when you copy and paste
+> solutions from StackOverflow. Ask @WeixuanZ to make sure you don't mistakenly
+> give root privileges to all files in `/usr/local/`.
```
Comment (tim-clifford) [inline README.md:37]: lol have you done this

**Review by tim-clifford — CHANGES_REQUESTED** [review]
Thanks very much for this! Just a few minor things

File: README.md [hunk 4]
```diff
@@ -13,9 +13,29 @@ your package manager...?
 See [the official download page](https://git-scm.com/download). The default
 options are fine. For these instructions to apply, launch Git Bash.
 
+A better solution is to install Windows Linux Subsystem (WSL), which is a full
+Linux kernel in a virtual machine. (If you think this is too much, just use Git Bash) 
+
+You may also be interested in downloading the _Windows Terminal_ from Microsoft Store.
+
```
Comment (tim-clifford) [inline README.md:20]: Slight reword so it doesn't sound like these instructions don't apply to WSL
```suggestion
The simplest solution is to use Git Bash, which you can get from [the official
download page](https://git-scm.com/download). The default options are fine.

A more extendible solution is to install Windows Subsystem for Linux (WSL),
which is a full Linux kernel in a virtual machine. (If you think this is too
much, just use Git Bash) 

You may also be interested in downloading the _Windows Terminal_ from the
Microsoft Store.
```

**Comment by WeixuanZ** [comment]
Changes look great

**Review by tim-clifford — APPROVED** [review]

**Pull request merged**